src/features/logging/instrument/index.js

```diff
@@ -0,0 +1,19 @@
+import { stringify } from '../../../common/util/stringify'
+import { InstrumentBase } from '../../utils/instrument-base'
+import { FEATURE_NAME } from '../constants'
+import { bufferLog } from '../shared/utils'
+
+export class Instrument extends InstrumentBase {
+  static featureName = FEATURE_NAME
+  constructor (agentIdentifier, aggregator, auto = true) {
+    super(agentIdentifier, aggregator, FEATURE_NAME, auto)
+
+    const instanceEE = this.ee
+    /** emitted by wrap-logger function */
+    this.ee.on('wrap-logger-end', function handleLog ([message, ...args]) {
+      const { level } = this
+      bufferLog(instanceEE, message, { ...(!!args.length && { customAttributes: stringify(args) }) }, level)
+    })
```

Comment:
So the wrappedFn executes, ends, passes information that it ended through the EE just for the logging instrument to put it back in the EE for later? This seems very roundabout to just having wrap-logger be
```
originalFn = function() {
  originalFn(...)
  bufferLog(...)
}
```

Reply:
Not really worried but it's very extraneous. Would it make more sense to have wrap-function capable of buffering it and just having a register handler in the aggregate for wrap-logger-end directly? For thought.